spec/std/crystal/hasher_spec.cr

```diff
@@ -57,9 +57,8 @@ describe "Crystal::Hasher" do
       Int64::MAX.hash.should eq(Int64::MAX.hash)
     end
 
-    # TODO: remove fencing when 128bits support is added to non-native platforms
```

Comment:
Why remove this comment?

Reply:
Because the fencing is indeed removed. The `flag?(:bits64)` is not part of it.

Reply:
Why not? I understand the comment translates to 32-bit platforms. And they are still fenced.